app/views/users/show.html.erb

```diff
@@ -4,16 +4,17 @@
   </div>
 </div>
 
-<div class="row mb-2">
-  <div class="col-md-6 offset-md-3">
-    <%= render "users/profile_nav", user: @user %>
-  </div>
-</div>
-
-<% @user.own_photos.each do |photo| %>
-  <div class="row mb-4">
+<% if policy(@user).show.photo? %>
```

Comment:
You are not calling this policy method correctly. Should be show_photo? or however you defined it in your user policy.